src/content/peerconnection/endtoend-encryption/css/main.css

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```

Comment:
the main.css file is copied from another source, so 2015 is right

Reply:
we should just bring those common things into main.css. Some day...

Reply:
i was tempted to change the css not to include audio but then I assume we'll add audio shortly :-)

Reply:
this is obsolete but github did not auto-resolve